### PR TITLE
task/3695 new design for hosting/my-virtual-machines

### PR DESCRIPTION
**Pull request by tiwariav**

### CODE REVIEW

**Comment by verysanghee** [comment]
screen shot 2017-08-16 at 18 16 55
some alignments needed and we are missing a bottom line of list. 
img-3934
the "view detail" btn is shown as default active btn on mobile.

**Comment by verysanghee** [comment]
img-3934
the "view detail" btn is shown as default active btn on mobile.

**Comment by verysanghee** [comment]
Tested and looks great

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -482,6 +482,12 @@ msgid ""
 "contact Data Center Light Support."
 msgstr ""
 
+#~ msgid "Ipv4"
```
Comment (tiwariav) [inline hosting/locale/de/LC_MESSAGES/django.po]: yes, removed both IPv4 and IPv6 intentionally, since they were abbreviations without translations.

**Pull request merged**